src/domains/Invite/InviteForm/InviteForm.tsx

```diff
@@ -0,0 +1,468 @@
+                            type='email'
+                            name='email input'
+                            aria-labelledby='email-input'
+                            color='primary'
+                        />
+                    </Grid>
+                </Collapse>
+            </Grid>
+            <Grid>
+                <FormControlLabel
+                    control={
+                        <Checkbox
+                            checked={showPreview}
+                            onChange={toggleShowPreview}
+                            name='show email preview checkbox'
+                            color='primary'
+                        />
+                    }
+                    label='show email preview'
+                />
+                <Collapse in={showPreview}>
+                    <div id='email-preview'>
+                        <EmailPreview
+                            fName='NAME_PLACEHOLDER'
+                            MoC={
+                                inviteForm.MoC ? inviteForm.MoC : 'Unknown MoC'
+                            }
+                            topic={
+                                inviteForm.topic
+                                    ? inviteForm.topic
+                                    : 'Unknown Topic'
+                            }
+                            eventDateTime={
+                                inviteForm.eventDateTime
+                                    ? inviteForm.eventDateTime
+                                    : 'Unknown Event Date'
+                            }
+                            constituentScope={inviteForm.constituentScope}
+                            registrationLink='https://connectingtocongress.org/register'
+                        />
+                    </div>
+                </Collapse>
+            </Grid>
+        </div>
+    );
+}
+
+interface NextStepAction {
+    type: 'next-step';
+}
+
+interface PreviousStepAction {
+    type: 'previous-step';
+}
+
+interface ResetAction {
+    type: 'reset-step';
+}
+
+type Actions = NextStepAction | PreviousStepAction | ResetAction;
+
+function stepReducer(step: number, action: Actions): number {
+    switch (action.type) {
+        case 'next-step':
+            if (step <= 4) return step + 1;
+            return step;
+        case 'previous-step':
+            if (step > 0) return step - 1;
+            return step;
+        case 'reset-step':
+            return 0;
+        default:
+            return step;
+    }
+}
+
+function getSteps() {
+    return [
+        'Select File',
+        'Verify Preview',
+        'Select Invite Delivery Date/Time',
+        'Invite Preview',
+        'Upload to Server',
+    ];
+}
+
+export default function InviteFormStepper() {
+    const classes = useStyles();
+    const [file, setFile]: [File | undefined, Function] = React.useState();
+    const [preview, setPreview]: [
+        object[] | undefined,
+        Function
+    ] = React.useState();
```

Comment:
I think this could be moved to the child components too.  With props added from my suggestions, such as `onFileSelected` or `onFileRemoved` we can then appropriately enable or disable the "next" button.